src/mongoose_maps.erl

```diff
@@ -0,0 +1,36 @@
+-module(mongoose_maps).
+-export([maps_append/3]).
+-export([maps_foreach/2]).
+-export([pairs_foreach/2]).
+-export([maps_or_pairs_foreach/2]).
+
+%% Appends a new Value to the current list of values associated with Key.
+maps_append(Key, Value, Map) ->
```

Comment:
Since this module is called `mongoose_maps`, I think we can skip `maps_` prefixes, like in case of `jid` module. :)

Reply:
prefixes are here, so we can import the functions into muc room module

Reply:
moved the functions into mongoose_lib.